src/diffusers/pipelines/auto_pipeline.py

```diff
@@ -0,0 +1,910 @@
+# coding=utf-8
+# Copyright 2023 The HuggingFace Inc. team.
+# Copyright (c) 2022, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
Why NVIDIA?

Reply:
I think I copied from the pipeline_utils.py

Reply:
@patrickvonplaten I think we can safely discard NVIDIA here no?